lib/screens/home_page.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
+import 'package:marvel/model/character.dart';
+import 'package:marvel/services/app_routes.dart';
+
+import 'package:marvel/services/network_service.dart';
+
+class HomePage extends StatelessWidget {
```

Comment:
HomePage is good name, but in this case I would reccommend calling this HerosPage to reflect the content.

Reply:
Done